setup.py

```diff
@@ -27,6 +27,11 @@ def read_version_py(infname):
         if mo:
             return mo.group(1)
 
+# make sure we have a proper version of python
+if sys.version_info[:2] != (2, 7):
+    raise RuntimeError("Python version 2.7 is required.")
+
```

Comment:
Thanks for the PR!
I don't think we actually depend on "2.7" (but definitely "less than 3.0" so maybe that would be a better check?)